ogm/releases/5.4/index.adoc

```diff
@@ -0,0 +1,35 @@
+:awestruct-layout: project-releases-series
+:awestruct-project: ogm
+:awestruct-series_version: "5.4"
+
+=== Core
+
+Hibernate OGM is now compatible with Hibernate ORM 5.3 and JPA 2.2!
+
+==== Components upgrade
+
+ * Hibernate ORM to 5.3.0.CR1
+ * Hibernate Search to 5.10.0.Beta2
+ * WildFly to 12.0.0
+ * Infinispan to 9.2.0.Final
+ * MongoDB to 3.6.3
+ * Neo4J to 3.3.5
+
+==== Module Distribution
+
+ * WildFly modules as feature pack distribution is now available
+
+==== Infinispan
+
+ * Caches created by the dialect are now transactional as default
+ * Caches created by the dialect could be configured using a cache configuration
+
+==== MongoDB
+
+ * Stored procedure with positional parameters in now supported
+ * Native query enhancement
+
+==== Fixes
+
+ * Bug fixes related to projections and native query pagination
```

Comment:
Aren't they already in the last 5.3? As we just started the 5.4 release cycle, I don't think we need to put them here.

Reply:
ok